deltachat-ios/AppDelegate.swift

```diff
@@ -61,9 +61,35 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         dcAccounts.logger = DcLogger()
         dcAccounts.openDatabase()
         migrateToDcAccounts()
+
+        if let sharedUserDefaults = UserDefaults.shared, !sharedUserDefaults.bool(forKey: UserDefaults.hasSavedKeyToKeychain) {
```

Comment:
so, `SecItem`s survives reinstall while `UserDefaults` does not? maybe add a more explicit note for that.

Reply:
that is the case, I'll add an comment